crates/runtime/src/starknet/context.rs

```diff
@@ -32,68 +28,28 @@ pub const SEQUENCER_ADDRESS: &str = "0x1000";
 pub const ERC20_CONTRACT_ADDRESS: &str = "0x1001";
 pub const STEP_RESOURCE_COST: f64 = 0.005_f64;
 
-// HOW TO FIND:
-// 1. https://docs.starknet.io/documentation/architecture_and_concepts/Network_Architecture/fee-mechanism/#calculation_of_computation_costs
+const CONSTANTS_13_0_JSON: &str = include_str!("./resources/versioned_constants_13_0.json");
+static DEFAULT_CONSTANTS: Lazy<VersionedConstants> = Lazy::new(|| {
+    serde_json::from_str(CONSTANTS_13_0_JSON).expect("Versioned constants JSON file is malformed")
+});
```

Comment:
Is it how they do it? 💀

Reply:
They do load a default one from a JSON file, but we currently need 0.13.0